src/ngraph/runtime/intelgpu/intelgpu_backend.cpp

```diff
@@ -0,0 +1,52 @@
+/*******************************************************************************
```

Comment:
This comment block should have all of the `*` at column zero. This is usually caused by an incorrect version for clang format. We use clang-format 3.9 (or 3.9.1 as I recall) to get consistent formatting. You will have to manually edit this as clang-format won't put it back they way we want it.